content/docs/modes/arp.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "ARP"
+weight: 21
+description: >
+  kube-vip ARP mode
+---
+
```

Comment:
Clarify in your parenthetical comment that this reference is only in context to kube-vip. ARP is a protocol which isn't really called layer 2 (an OSI model reference).

Second sentence avoid ending with prepositions:

"...so that a host knows to which physical or virtual NIC traffic should be sent."